### PR TITLE
feat(swag): add Devfolio opportunity

### LINKED ISSUE #717: DevFolio Hacktoberfest
## Wild swag opportunity appeared!

- [x] I've checked that this isn't a duplicate issue.

Q. Which company offers it?  
A. : DevFolio

Q. Where have you seen the announcement (URL)?  
A. : https://devfolio.co/blog/devfolio-hacktoberfest-2020/

Q. What kind of swag (clothing, stickers, ...)?  
A. : clothing, stickers

Q. Does it ship worldwide?  
A. : No, only India

- [x] I've checked that the promotion is legit.
- [x] I've checked that it's still available to the public.

### PR DESCRIPTION
**Pull request by achaljhawar**
Fixes #717

### CODE REVIEW

**Comment by achaljhawar** [comment]
@vikaspotluri123 is it okay now.

File: data.json [hunk 1]
```diff
@@ -718,5 +718,14 @@
     "image": "http://devblogs.microsoft.com/xamarin/wp-content/uploads/sites/44/2019/10/medium_25686-c9dd82896c9e66bb82600f256444abe6.png",
     "dateAdded": "2019-10-08T21:00:00.000Z",
     "tags": ["stickers", "hacktoberfest", "clothing", "expired"]
+  },
+  {
```
Comment (vikaspotluri123) [inline data.json:722]: Can you please put this in alphabetical order?

File: data.json [hunk 2]
```diff
@@ -718,5 +718,14 @@
...
+    "name":"DevFolio",
```
Comment (vikaspotluri123) [inline data.json:723]: Please update the formatting to follow the rest of the doc (space after colon (`:`))

File: data.json [hunk 3]
```diff
@@ -718,5 +718,14 @@
...
+    "difficulty":"medium",
+    "description":"Contribute to any of the following GitHub repositories in october metioned in the article to get stickers and swag from DevFolio shipping only in India.",
```
Comment (vikaspotluri123) [inline data.json:725]:
```suggestion
    "description": "Contribute 4 valid PRs to one of Devfolio's approved repositories, and receive some stickers! Top 50 contributors will also get a t-shirt.",
```

**Review by vikaspotluri123 — CHANGES_REQUESTED** [review]
Good start, have a few thing :)

**Comment by achaljhawar** [comment]
@vikaspotluri123 pls review it again

File: data.json [hunk 4]
```diff
@@ -187,6 +187,15 @@
     "image": "https://devfolio.sgp1.digitaloceanspaces.com/ghost/2019/10/ghost_download1.jpeg",
     "dateAdded": "2019-10-05T23:17:55.000Z",
     "tags": ["stickers", "clothing", "hacktoberfest", "expired"]
+  },
+    {
```
Comment (aslafy-z) [inline data.json:191]:
```suggestion
  {
```

File: data.json [hunk 5]
```diff
@@ -187,6 +187,15 @@
...
+    "name": "Devfolio",
+    "difficulty":"medium",
+    "description": "Contribute 4 valid PRs to one of Devfolio's approved repositories, and receive some stickers! Top 50 contributors will also get a t-shirt.",
+    "reference": "https://devfolio.co/blog/devfolio-hacktoberfest-2020/",
+    "image": "https://pbs.twimg.com/media/EJfpSDTUUAIXtdP?format=jpg&name=small",
```
Comment (aslafy-z) [inline data.json:196]:
```suggestion
    "image": "https://pbs.twimg.com/media/EJfpSDTUUAIXtdP.jpg",
```

File: data.json [hunk 6]
```diff
@@ -187,6 +187,15 @@
...
+    "dateAdded": "2020-10-01T16:35:21.000Z",
```
Comment (aslafy-z) [inline data.json:197]:
```suggestion
    "dateAdded": "2020-10-09T16:35:21.000Z",
```

**Review by aslafy-z — APPROVED** [review]
LGTM

**Comment by aslafy-z** [comment]
Closing in flavor of #732. I did not noticed this duplicate before, sorry for the effort.

**Pull request closed without merging**